packages/discord.js/typings/index.d.ts

```diff
@@ -320,6 +320,17 @@ export type GuildCacheMessage<Cached extends CacheType> = CacheTypeReducer<
   Message | APIMessage
 >;
 
+export interface InteractionResponses<Cached extends CacheType> {
```

Comment:
I'm not a fan of this name considering it's in plural. Also, doesn't it remove the other properties the object may have?

Reply:
By the way, this might need type tests so we can ensure the type is narrowed correctly.